zuul.d/jobs-thoth.yaml

```diff
@@ -25,6 +25,15 @@
           label: thoth-pytest
     run: playbooks/thoth/pytest.yaml
 
+- job:
+    name: thoth-pytest-py37
+    parent: base-openshift-pod
+    nodeset:
+      nodes:
+        - name: pod
+          label: thoth-pytest-py37
+    run: playbooks/thoth/pytest.yaml
```

Comment:
@harshad16 same playbook? I think it works, havent tested it.

Reply:
yes, we would run the pytest in a different environment.
Will change different playbook based upon feedbacks